.github/workflows/test.yml

```
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.5, 3.6, 3.7, 3.8, pypy3]
+        python-version: [3.6, 3.7, 3.8, pypy3]
```

3.9 is also available

```suggestion
        python-version: [3.6, 3.7, 3.8, 3.9, pypy3]
```

---

Yes, but tests still fail due to the urlparse changes mentioned #536